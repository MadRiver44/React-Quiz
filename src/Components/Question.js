import React from 'react';

const Question = props => {
  return (
    <div>
      <h1>{props.children}</h1>
    </div>
  );
};

export default Question;
