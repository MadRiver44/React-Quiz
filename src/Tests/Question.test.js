import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import Question from '../Components/Question';

// test handleclick
//

describe('Question component', () => {
  test('Question snapshot', () => {
    const tree = renderer.create(<Question question={'What is Redux?'} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('Question renders one question', () => {
    const container = document.createElement('div');
    ReactDOM.render(<Question question={'this is a question'} />, container);
    expect(container.textContent).toMatch('');
  });
});
