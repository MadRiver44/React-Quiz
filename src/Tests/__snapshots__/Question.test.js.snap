// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question component Question snapshot 1`] = `
<div>
  <h1 />
</div>
`;
